packages/cli/src/lib/implementation/validate-filter-options.utils.ts

```diff
@@ -1,81 +1,85 @@
-import type {
-  CategoryConfig,
-  CategoryRef,
-  PluginConfig,
-} from '@code-pushup/models';
+import type { PluginConfig } from '@code-pushup/models';
 import {
   capitalize,
   filterItemRefsBy,
   pluralize,
   ui,
 } from '@code-pushup/utils';
-import type {
-  FilterOptionType,
-  FilterOptions,
-  Filterables,
-} from './filter.model.js';
+import type { FilterOptionType, Filterables } from './filter.model.js';
 
 export class OptionValidationError extends Error {}
 
+// eslint-disable-next-line max-lines-per-function
 export function validateFilterOption(
   option: FilterOptionType,
   { plugins, categories = [] }: Filterables,
-  { itemsToFilter, verbose }: { itemsToFilter: string[]; verbose: boolean },
+  {
+    itemsToFilter,
+    skippedItems,
+    verbose,
+  }: { itemsToFilter: string[]; skippedItems: string[]; verbose: boolean },
 ): void {
   const itemsToFilterSet = new Set(itemsToFilter);
-  const validItems = isCategoryOption(option)
-    ? categories
-    : isPluginOption(option)
-      ? plugins
-      : [];
-  const invalidItems = itemsToFilter.filter(
-    item => !validItems.some(({ slug }) => slug === item),
+  const skippedItemsSet = new Set(skippedItems);
+  const validItemsSet = new Set(
+    isCategoryOption(option)
+      ? categories.map(({ slug }) => slug)
+      : isPluginOption(option)
+        ? plugins.map(({ slug }) => slug)
+        : [''],
```

Comment:
An array with an empty string? :raised_eyebrow: That looks like a mistake:

```suggestion
        : [],
```